### ISSUE TITLE
"dagger" category

### ISSUE THREAD

**Issue by DanGrayson** [UniMath/UniMath#1661]
The term "dagger category" is ridiculous, since the name is based on punctuation that is used in the notation.  What about "duality category", or "involutive category"?

@Kfwullaert

**Comment by DanGrayson** [comment]
<img width="825" alt="Screenshot 2023-03-17 at 9 21 09 AM" src="https://user-images.githubusercontent.com/700228/225961574-9443e779-0864-4cdf-bc08-2bca96058408.png">

**Comment by peterlefanulumsdaine** [comment]
The name may be unprincipled, but it’s concise, unambiguous, memorable, and very well-established.  (At least in my experience, it’s much more widely used than any of the mentioned alternatives.)

**Comment by arnoudvanderleer** [comment]
@benediktahrens @nmvdw What do you think of this proposal? Do you think it is wise to keep using the name dagger category, or do you think another name is preferred?

**Comment by benediktahrens** [comment]
I think it is best to stick with the established name, though I agree with Dan's objections. I am closing this issue.

**Comment by arnoudvanderleer** [comment]
Would it be an idea to add a note to the file, mentioning the (Dan's) objections?

**Comment by benediktahrens** [comment]
I don't think that UniMath is a good place to have a discussion on vocabulary used in CT.

**Comment by arnoudvanderleer** [comment]
Fair enough.
